packages/data/addon/serializers/dimensions/elide.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import NaviDimensionSerializer from './interface';
+import NaviDimensionModel from '../../models/navi-dimension';
+import { DimensionColumn } from '../../adapters/dimensions/interface';
+import { AsyncQueryResponse } from 'navi-data/adapters/facts/interface';
+import EmberObject from '@ember/object';
+
+type ResponseEdge = {
+  node: Dict<string>;
+};
+
+export default class ElideDimensionSerializer extends EmberObject implements NaviDimensionSerializer {
+  normalize(dimension: DimensionColumn, rawPayload: unknown): NaviDimensionModel[] {
+    const responseStr = (rawPayload as AsyncQueryResponse)?.asyncQuery.edges[0].node.result?.responseBody;
+    const { id: dimensionName, tableId } = dimension.columnMetadata;
```

Comment:
Do we need to strip off the table name from the id?

Reply:
I've updated the fact adapter to do that